src/utils/common.ts

```diff
@@ -71,6 +70,26 @@ export function max(a: BigNumberish, b: BigNumberish): BN {
 
 export const fixedPointAdjustment = toBNWei("1");
 
+/**
+ * Promote or demote a number to based on differing number of decimal places.
+ * @dev Not to be confused with a Web3 variant located in FormattingUtilsInternal.
+ */
+export const convertDecimals = (fromDecimals: number, toDecimals: number) => {
+  assert(fromDecimals >= 0, "requires fromDecimals as an integer >= 0");
+  assert(toDecimals >= 0, "requires toDecimals as an integer >= 0");
+  // amount: string, BN, number - integer amount in fromDecimals smallest unit that want to convert toDecimals
+  // returns: string with toDecimals in smallest unit
+  return (amount: BigNumberish): string => {
```

Comment:
Could we return this in terms of `BigNumberish` or directly as a `BigNumber`?

Reply:
I was able to drop this import after merging in your recent changes :+1: